frontend/src/Pages/HomePage/AcceptAssignmentPage.jsx

```diff
@@ -0,0 +1,40 @@
+import { Button, Divider, Modal, Space } from "antd";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+
+export function AcceptAssignmentPage() {
+  const navigate = useNavigate();
+
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const showModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const onCancel = () => {
+    navigate(-1);
+  };
+
+  return (
+    <>
+      <Modal open={showModal} closable={false} footer={false} width={400}>
```

Comment:
Set open={true} and remove the state because you never update that state after mounting this component.